package/tests/testNET.py

```diff
@@ -44,10 +44,10 @@ def test_runTestWithParams(self):
             self.assertEqual(prmon_rc, 0, "Non-zero return code from prmon")
             prmonJSON = json.load(open("prmon.json"))
             expectedBytes = 1025000 * requests * slack;
-            self.assertGreater(prmonJSON["Max"]["tot_rx_bytes"], expectedBytes, "Too low value for rx bytes "
-                               "(expected minimum of {0}, got {1})".format(expectedBytes, prmonJSON["Max"]["tot_rx_bytes"]))
-            self.assertGreater(prmonJSON["Max"]["tot_tx_bytes"], expectedBytes, "Too low value for tx bytes "
-                               "(expected minimum of {0}, got {1})".format(expectedBytes, prmonJSON["Max"]["tot_tx_bytes"]))
+            self.assertGreater(prmonJSON["Max"]["rx_bytes"], expectedBytes, "Too low value for rx bytes "
+                               "(expected minimum of {0}, got {1})".format(expectedBytes, prmonJSON["Max"]["rx_bytes"]))
+            self.assertGreater(prmonJSON["Max"]["tx_bytes"], expectedBytes, "Too low value for tx bytes "
+                               "(expected minimum of {0}, got {1})".format(expectedBytes, prmonJSON["Max"]["tx_bytes"]))
     
     return configurableProcessMonitor
 
```

Comment:
Here we use `urllib2` which makes this script python2 specific, if I'm not mistaken. In python3.6 it gives me:

```
Traceback (most recent call last):
  File "./netBurner.py", line 11, in <module>
    import urllib2
ModuleNotFoundError: No module named 'urllib2'
```

Also `CGIHTTPServer` is apparently moved into `http` module in python3. I just wanted to point out for future reference.

Reply:
Yeah, I really struggled to find a way to make this py2/3 neutral, but in the end I think I gave up and went for py2. The `six` module might save us:

https://pythonhosted.org/six/

Looking to the future, I would try to target python3, with python2 fallback.